pkgs/build-support/vm/default.nix

```diff
@@ -1,5 +1,14 @@
 { lib
+
+# pkgs used inside the VM
 , pkgs
+
+# pkgs used outside the VM
+# (similar to NixOS's `virtualisation.host.pkgs` option)
+# using over `buildPackages` as currently `nix-build -E '(import <nixpkgs> { }).pkgsCross.aarch64-multiplatform.buildPackages.qemu_kvm'`
+# fails on aarch64-darwin with `error: Don't know how to run aarch64-unknown-linux-gnu executables.`
+, hostPkgs ? pkgs.buildPackages
+
```

Comment:
Overall there are 3 packages which need to comre from `hostPkgs`:

 - `qemu_kvm`
 - `bash`  for `builder`
 - `coreutils` for `run-vm`